fab/fabfile.py

```diff
@@ -107,66 +109,49 @@ def _require_target():
 
 
 class DeployMetadata(object):
-    def __init__(self, env):
-        self.env = env
+    def __init__(self):
```

Comment:
good call. i hate the fabric `env` object. it's essentially inited and globally accessible always. do agree that passing in code_root, branch, and environment is the way to go